jekyll/_cci2/server/v4.1/air-gapped-installation/additional-considerations.adoc

```diff
@@ -101,3 +101,35 @@ kubectl patch svc circleci-proxy  -p '{"spec": {"type": "LoadBalancer", "externa
 ----
 
 Once complete, DNS records can be created for your server installation (server.internal.example.com) and (*.server.internal.example.com) for 10.0.0.5.
+
+[#tls-importing]
+== Importing Trusted TLS Certificates
+
+When using a GitHub Enterprise instance with self-signed or custom certificate authority certificates, CircleCI server can be configured to trust these certificates with two methods as described below.
+
+*Note*: Values provided for either of these methods are supported for GitHub Enterprise certificates only - no other application is supported at this time.
```

Comment:
This might need to be `No other application` - we only support GHE and that's it. We don't support self-signed certs on Vault, Postgres, or Mongo, for example.

Reply:
Agreed - we should make this no other application - removed VCS reference.